src/views/Team/Outlet.vue

```diff
@@ -2,97 +2,189 @@
-              </ion-chip>
-              <ion-chip v-else @click="selectBackground" title="upload background">
-                <ion-icon :icon="imageIcon" />
-                <ion-icon :icon="uploadIcon" />
-              </ion-chip>
+
+              <h2>Subteams</h2>
+              <ul v-if="subteams" class="subteams" data-cy="subteams">
+                <li v-for="t in subteams" :key="t.id">
+                  <router-link
+                    :to="{ name: 'ViewTeam', params: { teamSlug: t.slug } }"
+                  >
+                    <ion-chip outline>
+                      <avatar :profile="t" size="1.5em" />
+                      <div style="margin: 5px">{{ t.name }}</div>
+                    </ion-chip>
+                  </router-link>
+                </li>
+
+                <ion-chip outline>
+                  <ion-button
+                    data-cy="addSubTeamModal"
+                    v-if="canEdit"
+                    @click="intendToCreateSubTeam"
+                    size="small"
+                    fill="clear"
+                  >
+                    <ion-icon size="small" :icon="addIcon" />
+                  </ion-button>
+                </ion-chip>
+              </ul>
               <ion-button
                 data-cy-role="edit"
                 data-cy-edit-target="styles"
-                color="light"
+                color="primary"
                 @click="intendEditStyles"
                 size="small"
                 fill="clear"
+                style="margin-left: 5%"
               >
                 Edit Custom Styles
               </ion-button>
             </div>
-          </div>
-          <ion-toolbar>
-            <ion-button size="small" @click="showQr = !showQr" fill="clear">
-              <ion-icon size="small" :icon="qrCodeIcon" />
-            </ion-button>
-            <div slot="end">
-              <inline-link-list showTitle :items="footerLinks">
-                <li v-if="canEdit">
-                  <ion-button @click="intendEditFooterLinks" size="small" fill="clear">
-                    <ion-icon size="small" :icon="editIcon" />
-                  </ion-button>
-                </li>
-              </inline-link-list>
+
+            <!-- Div QRcode -->
+            <div v-if="state == 'qrcode'" class="ion-padding">
+              <qrcode
+                :text="fullLink"
+                :logo="logo"
+                :height="256"
+                :width="256"
+              />
+              <div v-for="l in socialLinks" :key="l.target">
+                <qrcode
+                  :text="l.target"
+                  :logo="getSocialIcon(l.platform)"
+                  class="socialLinks"
+                  :height="240 / socialLinks.length"
+                  :width="240 / socialLinks.length"
+                />
+              </div>
+            </div>
+
+            <!-- Div Feed -->
+            <div v-if="state == 'feed'">
```

Comment:
I'm wondering if, rather than having all the `div`s in here and switching between them via `v-if`, we might want to split these into separate views (and have an outlet here) that we can route to ... like `:slug/about`; `:slug/qr` ; `:slug/feed` etc ...

But maybe let's put that into a follow up ticket if this increased the work a lot now.

Reply:
I will create a ticket for this, I think it is not very well structured but it is working for now !!